### ISSUE TITLE
Unable to start selenoid on macOS

### ISSUE THREAD

**Issue by rosolko** [aerokube/selenoid#155]
Hi there, unfortunately I'm unable to start selenoid.

System: `macOS Sierra 10.12.5`
Docker: `version 17.06.0-ce, build 02c1d87`
Storage: `Driver: overlay2`

Script to start selenoid: `docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v ${HOME}:/root aerokube/cm:latest-release selenoid start --vnc --tmpfs 128`

Downloaded `docker images` images:
```bash
REPOSITORY          TAG                 IMAGE ID            CREATED             SIZE
selenoid/vnc        opera_46.0          fb9985c6b2d4        10 days ago         670MB
aerokube/cm         latest-release      4146595d92ca        2 weeks ago         18.1MB
aerokube/selenoid   latest-release      93ee78105aa5        2 weeks ago         12.9MB
selenoid/vnc        firefox_54.0        03220ba69562        2 weeks ago         607MB
selenoid/vnc        chrome_59.0         d40227600238        3 weeks ago         726MB
selenoid/vnc        opera_45.0          78bed2b7af9f        4 weeks ago         876MB
selenoid/vnc        chrome_58.0         74f4c083f379        4 weeks ago         663MB
selenoid/vnc        firefox_53.0        1d8481b8a181        6 weeks ago         726MB
```

Selenoid creates browser config file located by: `~/.aerokube/selenoid/browsers.json`
Logs from console: 
```bash
2017/07/07 12:18:33 Successfully saved configuration to /root/.aerokube/selenoid/browsers.json
2017/07/07 12:18:33 Starting Selenoid...
2017/07/07 12:18:34 Successfully started Selenoid
```
But `hub` isn't available(Unable to connect) on address: `http://localhost:4444/wd/hub`
Output from `docker ps` says: 
```bash
CONTAINER ID        IMAGE                              COMMAND                  CREATED              STATUS                          PORTS               NAMES
bfa75ee997ad        aerokube/selenoid:latest-release   "/usr/bin/selenoid..."   About a minute ago   Restarting (1) 45 seconds ago                       selenoid
```
Logs `docker logs selenoid` from container:
```bash
2017/07/07 12:18:34 Loading configuration files...
2017/07/07 12:18:34 /usr/bin/selenoid: browsers config: read error: open /etc/selenoid/browsers.json: no such file or directory
2017/07/07 12:18:36 Loading configuration files...
2017/07/07 12:18:36 /usr/bin/selenoid: browsers config: read error: open /etc/selenoid/browsers.json: no such file or directory
2017/07/07 12:18:38 Loading configuration files...
2017/07/07 12:18:38 /usr/bin/selenoid: browsers config: read error: open /etc/selenoid/browsers.json: no such file or directory
2017/07/07 12:18:40 Loading configuration files...
2017/07/07 12:18:40 /usr/bin/selenoid: browsers config: read error: open /etc/selenoid/browsers.json: no such file or directory
2017/07/07 12:18:42 Loading configuration files...
2017/07/07 12:18:42 /usr/bin/selenoid: browsers config: read error: open /etc/selenoid/browsers.json: no such file or directory
2017/07/07 12:18:44 Loading configuration files...
2017/07/07 12:18:44 /usr/bin/selenoid: browsers config: read error: open /etc/selenoid/browsers.json: no such file or directory
2017/07/07 12:18:48 Loading configuration files...
2017/07/07 12:18:48 /usr/bin/selenoid: browsers config: read error: open /etc/selenoid/browsers.json: no such file or directory
2017/07/07 12:18:56 Loading configuration files...
2017/07/07 12:18:56 /usr/bin/selenoid: browsers config: read error: open /etc/selenoid/browsers.json: no such file or directory
2017/07/07 12:19:09 Loading configuration files...
2017/07/07 12:19:09 /usr/bin/selenoid: browsers config: read error: open /etc/selenoid/browsers.json: no such file or directory
2017/07/07 12:19:35 Loading configuration files...
2017/07/07 12:19:35 /usr/bin/selenoid: browsers config: read error: open /etc/selenoid/browsers.json: no such file or directory
2017/07/07 12:20:27 Loading configuration files...
2017/07/07 12:20:27 /usr/bin/selenoid: browsers config: read error: open /etc/selenoid/browsers.json: no such file or directory
```
Thanks.

**Comment by vania-pooh** [comment]
@rosolko please check that you have /root/.aerokube/selenoid/browsers.json file on your Mac and it contains JSON. The issue can be also related to the fact that Mac is using a Linux virtual machine to start Docker.

**Comment by rosolko** [comment]
@vania-pooh Previously I sad that selenoid created this file and locate to that path.

**Comment by vania-pooh** [comment]
@rosolko ok. Then what is shown in `Binds` section of `# docker inspect selenoid`? There should be a volume specification.

**Comment by rosolko** [comment]
@vania-pooh 
```json
"Binds": [
    "/root/.aerokube/selenoid:/etc/selenoid:ro",
    "/var/run/docker.sock:/var/run/docker.sock"
]
```

**Comment by vania-pooh** [comment]
@rosolko ok, the issue is in this line: `/root/.aerokube/selenoid:/etc/selenoid:ro`. `cm` in Docker container is using home from container to mount. Created https://github.com/aerokube/cm/issues/50. To make it work for now just use a [binary](https://github.com/aerokube/cm/releases/latest) instead of container.

**Comment by rosolko** [comment]
@vania-pooh Please with updates related to this issue. Thanks.

**Comment by vania-pooh** [comment]
@rosolko meanwhile does it work with binary?

**Comment by rosolko** [comment]
@vania-pooh Yes. Selenoid started and show me default browsers.
```bash
curl http://localhost:4444/status
$ {"total":5,"used":0,"queued":0,"pending":0,"browsers":{"chrome":{"58.0":{},"59.0":{}},"firefox":{"53.0":{},"54.0":{}},"opera":{"45.0":{},"46.0":{}}}}
```
Also `selenoid-ui` works well with binary.

**Comment by rosolko** [comment]
@vania-pooh But. If looks like loading default configuration of browsers isn't work well related to logs:
```bash
$ docker logs selenoid
2017/07/07 14:25:04 Loading configuration files...
2017/07/07 14:25:04 Loaded configuration from [/etc/selenoid/browsers.json]
2017/07/07 14:25:04 Using default containers log configuration because of: read error: open config/container-logs.json: no such file or directory
2017/07/07 14:25:04 Timezone: UTC
2017/07/07 14:25:04 Listening on :4444
```

**Comment by vania-pooh** [comment]
```
2017/07/07 14:25:04 Using default containers log configuration because of: read error: open config/container-logs.json: no such file or directory
```
This is the expected behavior. Need to fix the message.

**Comment by vania-pooh** [comment]
@aandryashin this warning with log configuration file is considered as an error. However that's not an error.

**Comment by vania-pooh** [comment]
Same stuff with Windows for the same reason.

**Comment by vania-pooh** [comment]
This is confirmed to be fixed in `cm` version 1.2.2+ and should now work when you update cm container.
